src/components/featured-maps-list/featured-maps-list-styles.module.scss

```diff
@@ -0,0 +1,53 @@
+@import 'styles/settings';
+@import 'styles/typography-extends';
+@import 'styles/ui.module';
+
+$section-padding: 0 24px;
+
+.cardsContainer {
+  display: flex;
+  flex-direction: column;
+  width: $sidebar-width;
+  border-radius: 2px;
+  transition: all 0.2s ease-out;
+}
+
+.scrollable {
```

Comment:
I think you can get rid of this `scrollable` `class` and just add `@extend %verticalScrollBar;` to the `cardsContainer`.

Reply:
fun fact: without the additional `.scrollable` class (moving the scroll to cardsContainer) the blur effect stops working 🤷‍♀ so I'll extend the verticalScrollbar in the scrollable class